packages/core/src/lib/filter/filter_rpc.ts

```diff
@@ -8,16 +8,16 @@ export type ContentFilter = {
 /**
  * FilterRPC represents a message conforming to the Waku Filter protocol
  */
-export class FilterRPC {
-  public constructor(public proto: proto.FilterRPC) {}
+export class FilterRpc {
```

Comment:
Why did you change capitalisation here?

Reply:
Capitalization was changed on proto file to proper PascalCase, I just match it as PascalCase is what we use across the codebase.